edgedb/__init__.py

```diff
@@ -110,6 +110,7 @@
     QueryArgumentError,
     MissingArgumentError,
     UnknownArgumentError,
+    InvalidArgumentValueError,
     NoDataError,
 )
-# </ERRORS-AUTOGEN>
+# </ERRORS-AUTOGEN>
```

Comment:
Can you fix the tool to add that newline if needed?

Reply:
Fixed